data/conky-dfly-smp.conf

```diff
@@ -36,7 +39,8 @@ conky.config = {
     extra_newline = false,
     own_window = false,
     own_window_class = 'Conky',
-    own_window_type = 'desktop',
+    own_window_type = 'normal',
+    own_window_hints = 'undecorated,sticky,below,skip_taskbar,skip_pager',
```

Comment:
Changed this to `normal` because it's the type that most commonly works, on WMs/DEs that are most commonly used (with desktops).